src/main/java/org/roaringbitmap/RoaringArray.java

```diff
@@ -578,4 +578,30 @@ protected int size() {
   public void writeExternal(ObjectOutput out) throws IOException {
     serialize(out);
   }
+
+  /**
+   * Gets the first value in the array
+   * @return te first value in the array
+   */
+  public int first() {
+    if(size == 0) {
+      return 0;
```

Comment:
I agree. I'm not sure if Collections::max relates to this functionality, but SortedSet::first, SortedSet::last throw NoSuchElementException for empty sets. I must admit until now I had only considered bitmaps containing natural numbers, but to be consistent with the comments on the PositiveIterator thread the only option would be to throw.